src/common/types/rsvp.ts

```
@@ -93,42 +40,226 @@ export const rsvpConfigSchema = z
-      example: true,
-    }),
-    createdAt: z.number().int().meta({
-      description: "Epoch timestamp (sec) when the RSVP was created.",
-      example: 1705512000,
-    }),
-  })
-  .meta({
-    description: "Represents a single confirmed RSVP record in the database.",
-    id: "RsvpItem",
-  });
+export const rsvpItemSchema = z.object({
```

ESLint

[error] Insert `⏎··`

(prettier/prettier)